mopidy_gmusic/library.py

```diff
@@ -311,14 +313,30 @@ def sorter(track):
     def refresh(self, uri=None):
         self.tracks = {}
         self.albums = {}
+        self.albums2 = {}
         self.artists = {}
+        self.artists2 = {}
 
         album_tracks = {}
         for track in self.backend.session.get_all_songs():
+            
             mopidy_track = self._to_mopidy_track(track)
 
             self.tracks[mopidy_track.uri] = mopidy_track
-            self.albums[mopidy_track.album.uri] = mopidy_track.album
+
+#           Need another albums list that will store all albumIds so they can be referenced in the browsing later
+#           Problem caused by Google assigning multiple albumIds for the same album
+
+            self.albums2[mopidy_track.album.uri] = mopidy_track.album
+            album_found = False
+            if len(self.albums) > 0:
```

Comment:
I would just drop this if-else and use the for loop all the time. Looping over an empty list shouldn't be very expensive.

Reply:
OK, sure